apps/journeys-admin/src/components/JourneyList/JourneyStatusFilter/JourneyStatusFilter.tsx

```diff
@@ -0,0 +1,52 @@
+import Tune from '@mui/icons-material/Tune'
+import { useTranslation } from 'next-i18next'
+import { ReactElement } from 'react'
+
+import { useBreakpoints } from '@core/shared/ui/useBreakpoints'
+
+import { RadioSelect, RadioSelectOption } from '../RadioSelect'
+
+import type { JourneyStatus } from '../JourneyListView/JourneyListView'
+
+interface JourneyStatusFilterProps {
+  status?: JourneyStatus
+  onChange: (value: JourneyStatus) => void
+  open?: boolean
+}
+
+export function JourneyStatusFilter({
+  status,
+  onChange,
+  open
+}: JourneyStatusFilterProps): ReactElement {
+  const { t } = useTranslation('apps-journeys-admin')
+  const breakpoints = useBreakpoints()
+
+  const statusOptions: RadioSelectOption<JourneyStatus>[] = [
+    {
+      value: 'active',
+      label: t('Active')
+    },
+    {
+      value: 'archived',
+      label: t('Archived')
+    },
+    {
+      value: 'trashed',
+      label: t('Trash')
+    }
+  ]
+
+  return (
+    <RadioSelect
+      value={status}
+      defaultValue="active"
+      options={statusOptions}
+      onChange={onChange}
+      ariaLabel={t('Filter by status')}
+      open={open}
+      sx={{ marginRight: breakpoints.sm ? '12px' : '4px' }}
```

Comment:
[See earlier comment on using spacing tokens instead of pixels](https://github.com/JesusFilm/core/pull/8473/files#r2600877999)

```suggestion
      sx={{ mr: breakpoints.sm ? 3 : 1 }}
```